src/components/Footer.vue

```diff
@@ -37,7 +40,8 @@ export default {
     data() {
         return {
             licensePoliciesUrl: 'https://creativecommons.org/policies#license',
-            CCBYUrl: 'https://creativecommons.org/licenses/by/4.0/'
+            CCBYUrl: 'https://creativecommons.org/licenses/by/4.0/',
```

Comment:
Not a big deal, but why not put these URLs directly into the href attributed on the link elements themselves? AFAIK, these links are static.

Reply:
I put them separately because the links become a part of translated text of the footer. So, I wanted to keep the text clear of unnecessary and potentially confusing details for translators.

We could also not extract any strings in the footer because it would eventually be replaced by the Vocabulary footer and be translated then.

Reply:
I looked at the other strings: we have links in them, too, so I just added them in.